misk-rate-limiting-bucket4j-dynamodb-v1/src/test/kotlin/misk/ratelimiting/bucket4j/dynamodb/v1/DynamoDBV1RateLimiterTest.kt

```diff
@@ -0,0 +1,113 @@
+package misk.ratelimiting.bucket4j.dynamodb.v1
+
+import com.google.inject.Module
+import com.google.inject.Provides
+import io.micrometer.core.instrument.Clock
+import io.micrometer.core.instrument.MeterRegistry
+import io.micrometer.prometheus.PrometheusConfig
+import io.micrometer.prometheus.PrometheusMeterRegistry
+import io.prometheus.client.CollectorRegistry
+import jakarta.inject.Inject
+import jakarta.inject.Singleton
+import misk.MiskTestingServiceModule
+import misk.aws.dynamodb.testing.DockerDynamoDbModule
+import misk.aws.dynamodb.testing.DynamoDbTable
+import misk.environment.DeploymentModule
+import misk.inject.KAbstractModule
+import misk.testing.MiskTest
+import misk.testing.MiskTestModule
+import misk.time.FakeClock
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import wisp.deployment.TESTING
+import wisp.ratelimiting.RateLimiter
+import wisp.ratelimiting.testing.TestRateLimitConfig
+
+@MiskTest(startService = true)
+class DynamoDBV1RateLimiterTest {
+  @Suppress("unused")
+  @MiskTestModule
+  private val module: Module = object: KAbstractModule() {
+    override fun configure() {
+      install(
+        DockerDynamoDbModule(
+          DynamoDbTable(DyRateLimitBucket::class)
+        )
+      )
+      install(DynamoDbV1Bucket4jRateLimiterModule("rate_limit_buckets"))
+      install(MiskTestingServiceModule())
+      install(DeploymentModule(TESTING))
+    }
+
+    @Provides @Singleton
+    // In prod this is provided by Skim
+    fun provideMeterRegistry(collectorRegistry: CollectorRegistry): MeterRegistry {
+      return PrometheusMeterRegistry(
```

Comment:
Would recommend using `SimpleMeterRegistry` if you don't need to test anything prometheus specific.

Reply:
Thanks, fixed this everywhere I added it 🙏